01_fib/baijie/fib_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := strconv.Quote("1\n")
```

Comment:
You also want to change this to test for the output on the spec.

Reply:
Done